cmd/kubeadm/app/phases/controlplane/manifests_test.go

```diff
@@ -59,26 +60,45 @@ func TestGetStaticPodSpecs(t *testing.T) {
 		}},
 	}
 
+	apiserverDefaultPort := int32(6443)
 	// Executes GetStaticPodSpecs
-	specs := GetStaticPodSpecs(cfg, &kubeadmapi.APIEndpoint{}, []kubeadmapi.EnvVar{})
+	specs := GetStaticPodSpecs(cfg, &kubeadmapi.APIEndpoint{
+		BindPort: apiserverDefaultPort,
+	}, []kubeadmapi.EnvVar{})
 
 	var tests = []struct {
-		name          string
-		staticPodName string
-		env           []v1.EnvVar
+		name                 string
+		staticPodName        string
+		expectLivenessProbe  bool
+		expectReadinessProbe bool
+		expectStartupProbe   bool
+		probePort            int32
+		env                  []v1.EnvVar
 	}{
 		{
-			name:          "KubeAPIServer",
-			staticPodName: kubeadmconstants.KubeAPIServer,
+			name:                 "KubeAPIServer",
+			staticPodName:        kubeadmconstants.KubeAPIServer,
+			expectLivenessProbe:  true,
+			expectReadinessProbe: true,
+			expectStartupProbe:   true,
+			probePort:            apiserverDefaultPort,
```

Comment:
there is a constant for that in constants.go

Reply:
> there is a constant for that in constants.go

no, there have not a constant for it, this is the reason i make it.

i updated to using `v1beta4.DefaultAPIBindPort`, how aboud now.